src/back/msl/writer.rs

```diff
@@ -1967,17 +2012,27 @@ impl<W: Write> Writer<W> {
                             names: &self.names,
                             usage: GlobalUse::empty(),
                             access: crate::StorageAccess::empty(),
-                            first_time: false,
+                            first_time: true,
```

Comment:
why is this `true` now?

Reply:
Because we want to have `float` here, instead of `type6` which is the alias that gets used every time after the first.

Reply:
what happens if we let it use `type6`?

Reply:
We get the error:
```
testfrag.metal:40:30: error: type 'type6 [1]' is not valid for attribute 'clip_distance'
    type6 gl_ClipDistance1 [[clip_distance]] [1];
                             ^~~~~~~~~~~~~
testfrag.metal:42:8: error: invalid return type 'main2Output' for vertex function
vertex main2Output main2(
       ^
2 errors generated.
```

Reply:
ok, cool, thanks!